Probably a useful patch given that we're catching errors:
```
/__w/dealii/dealii/include/deal.II/numerics/error_estimator.templates.h:380:53: error: cannot convert ‘dealii::Tensor<0, 3, std::complex<double> >::tensor_type’ {aka ‘std::complex<double>’} to ‘dealii::AlignedVector<double>::value_type’ {aka ‘double’} in assignment
  380 |             (parallel_data.psi(n, point)[component] *
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
      |                                                     |
      |                                                     dealii::Tensor<0, 3, std::complex<double> >::tensor_type {aka std::complex<double>}
  381 |              parallel_data.normal_vectors[point]);
      |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~    
/__w/dealii/dealii/include/deal.II/numerics/error_estimator.templates.h:391:50: error: no match for ‘operator+=’ (operand types are ‘dealii::AlignedVector<double>::value_type’ {aka ‘double’} and ‘dealii::Tensor<0, 3, std::complex<double> >::tensor_type’ {aka ‘std::complex<double>’})
  391 |               parallel_data.phi(n, p, component) +=
      |               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
  392 |                 (parallel_data.neighbor_psi(n, p)[component] *
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  393 |                  parallel_data.neighbor_normal_vectors[p]);
      |                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/dealii/dealii/include/deal.II/numerics/error_estimator.templates.h:450:48: error: no match for ‘operator-=’ (operand types are ‘dealii::AlignedVector<double>::value_type’ {aka ‘double’} and ‘__gnu_cxx::__alloc_traits<std::allocator<std::complex<double> >, std::complex<double> >::value_type’ {aka ‘std::complex<double>’})
  450 |                 parallel_data.phi(n, point, 0) -=
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
  451 |                   parallel_data.neumann_values1[point];
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/dealii/dealii/include/deal.II/numerics/error_estimator.templates.h:465:58: error: no match for ‘operator-=’ (operand types are ‘dealii::AlignedVector<double>::value_type’ {aka ‘double’} and ‘std::complex<double>’)
  465 |                   parallel_data.phi(n, point, component) -=
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
  466 |                     parallel_data.neumann_values[point](component);
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -135,6 +135,7 @@ jobs:
               -D DEAL_II_CXX_FLAGS='-Werror -std=c++20 ${{ matrix.flags }}' \
               -D DEAL_II_EARLY_DEPRECATIONS=ON \
               -D DEAL_II_COMPONENT_PYTHON_BINDINGS=ON \
+              -D DEAL_II_WITH_COMPLEX_VALUES="ON" \
               -D DEAL_II_WITH_MPI="ON" \
               -D DEAL_II_WITH_CGAL="ON" \
               -D DEAL_II_WITH_HDF5="ON" \

